check.go

```diff
@@ -57,6 +57,22 @@ Loop:
 			}
 		}
 
+		if len(request.Source.StatusFilters) > 0 {
+			for _, statusFilter := range request.Source.StatusFilters {
+				isValid := false
+				for _, prStatus := range p.Tip.Status.Contexts {
+					if prStatus.Context == statusFilter.Context && strings.EqualFold(prStatus.State, statusFilter.State) {
+						// requires both that status and that that status matches the desired state
```

Comment:
is this saying requires that there is a status?

Reply:
Yeah, I debated this, but I think if you add one of these status filters, then it should also reject anything without that status

Reply:
gotcha, thoughts on this rephrasing?

```suggestion
						// requires the given status exists and that it matches the desired state
```